src/main/msp/msp.c

```diff
@@ -2522,7 +2522,7 @@ static mspResult_e mspFcProcessOutCommandWithArg(mspDescriptor_t srcDesc, int16_
             // type byte, then length byte followed by the actual characters
             const uint8_t textType = sbufBytesRemaining(src) ? sbufReadU8(src) : 0;
 
-            char* textVar;
+            const char* textVar;
```

Comment:
```suggestion
            const char *textVar;
```

Reply:
Original it should be:

    char const* textVar;

https://qr.ae/prygdq